java/src/jmri/PowerManager.java

```diff
@@ -50,6 +51,8 @@
     public void addPropertyChangeListener(@CheckForNull PropertyChangeListener p);
 
     public void removePropertyChangeListener(PropertyChangeListener p);
+    
+    public boolean implementsIdle();
```

Comment:
To avoid having to pull those implementations in, and to make this more readable, please put something like:
```
    /**
      * Returns false to indicate it does not implement an "IDLE" power state.
      * @return false
      */
     public default boolean implementsIdle() {
         return false;
     }
```
and then add some discussion of the IDLE state to the Javadoc.  This is the class that defines PowerManager so it needs to have the most complete documentation of it.

Reply:
Ah, "default" may be the piece I was missing, which caused me to implement the various routines in a bunch of places I thought I would not have to touch!  Will re-work and re-submit.

Reply:
"Default" is now implemented.

Reply:
In addition to the implementation of the default, which you have now put in place, you now need to remove the duplicative versions of this method in all the power managers that are just returning false.

Reply:
Done.

Reply:
A question of protocol: I believe that the issues behind this change request have been resolved.  Must @bobjacobsen "resolve the conversation" related to this "change request"?

Reply:
Normally, when your commits in this area would have cleared that.  But for some reason it didn't this time, so I'll clear it manually.  Thanks for the reminder.